templates/partials/navigation.phtml

```diff
@@ -19,40 +20,37 @@ declare(strict_types=1);
         <div class="collapse navbar-collapse" id="navbarCollapse">
             <ul class="navbar-nav ml-auto">
                 <li class="nav-item dropdown">
-                    <a class="nav-link dropdown-toggle<?= str_starts_with($this->url(), $this->url('about.overview')) || str_starts_with($this->url(), $this->url('security')) ? ' active' : '' ?>"
+                    <a class="nav-link dropdown-toggle<?= str_starts_with($currentUrl, $this->url('about.overview')) || str_starts_with($currentUrl, $this->url('security')) ? ' active' : '' ?>"
                        href="#"
                        data-toggle="dropdown"
                        aria-expanded="false">About</a>
                     <div class="dropdown-menu">
-                        <a class="dropdown-item<?= $this->url() === $this->url('about.overview') ? ' active' : '' ?>" href="<?= $this->url('about.overview') ?>">Overview</a>
-                        <a class="dropdown-item<?= $this->url() === $this->url('about.tsc') || $this->url() === $this->url('about.join') ? ' active' : '' ?>" href="<?= $this->url('about.tsc') ?>">Technical Steering Committee</a>
-                        <a class="dropdown-item<?= $this->url() === $this->url('about.foundation') ? ' active' : '' ?>" href="<?= $this->url('about.foundation') ?>">About the Foundation</a>
-                        <a class="dropdown-item<?= str_starts_with($this->url(), $this->url('security')) ? ' active' : '' ?>" href="<?= $this->url('security') ?>">Security</a>
+                        <a class="dropdown-item<?= $currentUrl === $this->url('about.overview') ? ' active' : '' ?>" href="<?= $this->url('about.overview') ?>">Overview</a>
```

Comment:
Thanks for the suggestions, just uploaded a version using the ServerUrlHelper which doesn't need the try-catch at all. I have found another small issue though, because of `$this->url()` adding a trailing slash.

For urls with optional slashes, such as https://getlaminas.org/about[/], the "active" class will not be added to the link if the trailing slash is omitted in the url. This could be fixed by simply trimming the trailing slashes for on all checks with `$this->url()` but that will add a bit of clutter, or maybe replacing them with a separate function call that handles this?

 What would you prefer?

Reply:
I was actually commenting on that. See the suggestion in comment above. I think closure approach will make template a bit cleaner

Reply:
This whole approach to mark active nav elements is a hack. It can work only for the literal routes and won't work any parametrized routes.

For anyone looking at this: no, this is not how you should do it in you application :laughing:

Reply:
> This whole approach to mark active nav elements is a hack.
> …
> For anyone looking at this: no, this is not how you should do it in you application 😆

Correct!